packages/graphiql/src/components/GraphiQL.tsx

```diff
@@ -1195,12 +1182,24 @@ export class GraphiQL extends React.Component<GraphiQLProps, GraphiQLState> {
           docExplorerOpen: true,
           docExplorerWidth: Math.min(docsSize, 650),
         });
+        this._storage.set(
+          'docExplorerWidth',
+          JSON.stringify(this.state.docExplorerWidth),
```

Comment:
ah, another one that needs debounced. anything that has to do with high throughput events